src/fft_fftw.cpp

```diff
@@ -179,8 +179,7 @@ void Fftw::inverse(ComplexMat &complex_input, MatScales &real_result)
     fftwf_complex *in = reinterpret_cast<fftwf_complex *>(complex_input.get_p_data());
     float *out = real_result.ptr<float>();
 
-    if (n_channels == 1)
-        fftwf_execute_dft_c2r(plan_i_1ch, in, out);
+    if (n_channels == 1) fftwf_execute_dft_c2r(plan_i_1ch, in, out);
```

Comment:
Do not put this on a single line. I've changed the clang-forrmat configuration. Configure your tools to use the configuration from the repository.